packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/flush-archive/flush-archive.html

```diff
@@ -0,0 +1,29 @@
+<oui-modal
+    data-heading="{{ $ctrl.getHeadingContent() | translate }}"
+    data-primary-action="$ctrl.flushArchive()"
+    data-primary-label="{{:: 'pci_projects_project_storages_containers_container_cold_archive_flush_submit_label' | translate }}"
+    data-primary-disabled="$ctrl.isLoading || !$ctrl.terminateConfirmation || $ctrl.container.status === $ctrl.COLD_ARCHIVE_CONTAINER_STATUS.NONE"
```

Comment:
**nitpick:** Button should be disabled if the data entered is not valid with respect to `ng-pattern`

Reply:
Yes, `!$ctrl.terminateConfirmation` is the attribute that is related to the `ng-pattern` check.